miner/worker.go

```diff
@@ -819,6 +831,30 @@ func (w *worker) commitTransaction(env *environment, tx *types.Transaction) ([]*
 	}
 	env.txs = append(env.txs, tx)
 	env.receipts = append(env.receipts, receipt)
+	if w.config.Mev.MevEnabled {
+		minerFee, _ := tx.EffectiveGasTip(env.header.BaseFee)
+		env.profit.Add(env.profit, new(big.Int).Mul(new(big.Int).SetUint64(receipt.GasUsed), minerFee))
+	}
+	return receipt.Logs, nil
+}
+
+func (w *worker) commitBundleTransaction(env *environment, tx *types.Transaction, unRevertible bool) ([]*types.Log, error) {
+	if tx.Type() == types.BlobTxType {
+		return w.commitBlobTransaction(env, tx)
+	}
+	receipt, err := w.applyTransaction(env, tx)
+	if err != nil {
+		return nil, err
+	}
+	if receipt.Status == types.ReceiptStatusFailed && unRevertible {
+		return nil, errors.New("no revertible transaction failed")
+	}
+	env.txs = append(env.txs, tx)
+	env.receipts = append(env.receipts, receipt)
+	if w.config.Mev.MevEnabled {
+		minerFee, _ := tx.EffectiveGasTip(env.header.BaseFee)
+		env.profit.Add(env.profit, new(big.Int).Mul(new(big.Int).SetUint64(receipt.GasUsed), minerFee))
```

Comment:
Is it possible to also consider the L1 fee in the profit?

Reply:
I think our L1 fee is not used as profit, but to cover the fees we pay when submitting transactions to L1. If L1 fee is taken into consideration, it is not easy for us to simply add it directly, because we also need to consider the actual payment of L1.

Reply:
I probably misunderstood what you meant. It should mean that when adding the each transaction fee, for a transaction with a gas price of 0, the fee of the transaction is deducted by l1 fee and then added as profit, right?

Reply:
If it it, I think it is reasonable and feasible, but there may be a problem here. The profit we ultimately consider here is the entire worker, which may include multiple bundles. There is the possibility that some bundles may be paid for by other bundles. , the bundle gas price floor is still needed.